src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -358,99 +363,97 @@ export const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ ne
                     iconType: 'error',
                   })
                 }
-                ref={coverDialogRef}
+                ref={avatarDialogRef}
               />
             </>
           )}
         />
 
-        <StyledTitleSection className={transitions.names.slide}>
-          <Controller
-            name="avatar"
-            control={control}
-            render={() => (
-              <>
-                <StyledAvatar
-                  assetUrl={avatarAsset?.url}
-                  hasAvatarUploadFailed={hasAvatarUploadFailed}
-                  size="fill"
-                  onEditClick={() => avatarDialogRef.current?.open()}
-                  editable
-                  loading={loading}
-                />
-                <ImageCropDialog
-                  imageType="avatar"
-                  onConfirm={handleAvatarChange}
-                  onError={() =>
-                    displaySnackbar({
-                      title: 'Cannot load the image. Choose another.',
-                      iconType: 'error',
-                    })
-                  }
-                  ref={avatarDialogRef}
-                />
-              </>
-            )}
-          />
-
-          <TitleContainer>
-            {!loading || newChannel ? (
-              <>
-                <Controller
-                  name="title"
-                  control={control}
-                  rules={textFieldValidation({ name: 'Channel name', minLength: 3, maxLength: 40, required: true })}
-                  render={({ field: { ref, value, onChange } }) => (
-                    <Tooltip text="Click to edit channel title">
-                      <StyledHeaderTextField
-                        ref={ref}
-                        placeholder="Channel title"
-                        value={value}
-                        onChange={(e) => {
-                          onChange(e.currentTarget.value)
-                        }}
-                        error={!!errors.title}
-                        helperText={errors.title?.message}
-                      />
-                    </Tooltip>
-                  )}
-                />
-                {!newChannel && (
-                  <StyledSubTitle>{channel?.follows ? formatNumberShort(channel.follows) : 0} Followers</StyledSubTitle>
+        <TitleContainer>
+          {!loading || newChannel ? (
+            <>
+              <Controller
+                name="title"
+                control={control}
+                rules={textFieldValidation({ name: 'Channel name', minLength: 3, maxLength: 40, required: true })}
+                render={({ field: { ref, value, onChange } }) => (
+                  <Tooltip text="Click to edit channel title">
+                    <StyledHeaderTextField
+                      ref={ref}
+                      placeholder="Channel title"
+                      value={value}
+                      onChange={(e) => {
```

Comment:
Meaningful name please
```suggestion
                      onChange={(event) => {
```